src/cli/input_handler.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019 Luxoft Sweden AB
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#ifndef CONNECTIVITY_MANAGER_CLI_INPUT_HANDLER_H
```

Comment:
Not standard.

In practice, it is available everywhere we care about. I actually thought about using it when starting writing this code (and later again when moving some of it out to the D-Bus template) since I know it has been something practiced in previous projects here. But, decided against it since it is mentioned that it should not be used in some guidelines that @JEderonn wanted to use.